GCC_2019_07_10/dbplyr lightning talk/01 set up.R

```diff
@@ -7,7 +7,6 @@ install.packages("dbplyr") # Needs to be installed
 #  Load Libraries
 library(odbc)
 library(dplyr) # We only need to load dplyr (not dbplyr)
-library(magrittr)
```

Comment:
Sounds good to me - don't know much about it so I'm ok. If you could add some explanation about 'reflexive pipe' for less knowledgeable analysts like me, that'd be helpful.
PS: Is **reflexive pipe** something about a variable being assigned the result of a function applied to the variable itself?

Reply:
I think I made that name up... it's actually the [assignment pipe](https://magrittr.tidyverse.org/reference/compound.html)

This `x <- x %>% nrow()` is equivalent to `x %<>% nrow()` but you need to load magrittr to do it. It's not really necessary and if it's confusing then it's detracting from what I was trying to show!